### PR TITLE
Upgrade to Kotlin version (3.20) and fix any issues.

### PR DESCRIPTION
**Pull request by pueffl**
- Migrate to Kotlin 3.20
- Remove Kotlin Experimentals
- Migrate Kotlin Coroutines

### CODE REVIEW

File: app/ui/build.gradle [hunk 1]
```diff
@@ -41,7 +41,8 @@ dependencies {
     implementation "net.jcip:jcip-annotations:1.0"
     implementation "com.jakewharton.timber:timber:${versions.timber}"
     implementation "org.apache.james:apache-mime4j-core:${versions.mime4j}"
-    implementation "org.jetbrains.anko:anko-coroutines:0.10.4"
+    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.1.0"
+    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:1.1.0"
```
Comment (dri94) [inline app/ui/build.gradle]: 1.1.1 is out

File: build.gradle [hunk 2]
```diff
@@ -7,7 +7,7 @@ buildscript {
         ]
 
         versions = [
-                'kotlin': '1.2.71',
+                'kotlin': '1.3.20',
```
Comment (dri94) [inline build.gradle]: 1.3.21 is out

File: app/ui/build.gradle (hunk 1 above)
Comment (pueffl) [reply app/ui/build.gradle]: I know, but this pull reguest is open for a while. We can change this easily.

File: build.gradle (hunk 2 above)
Comment (pueffl) [reply build.gradle]: Same as above :-)

**Comment by cketti** [comment]
Thanks!

Sadly, this doesn't really make use of the new features of the new coroutines API. But I guess we can improve upon this in a future pull request.

**Pull request merged**